src/main/java/io/stargate/sgv2/jsonapi/StargateJsonApi.java

```diff
@@ -324,25 +324,33 @@
                         "data": {
                           "docs": [
                             {
-                               "_id": "1",
-                               "location": "New York",
-                               "race": {
-                                 "competitors": 100,
-                                 "start_date": "2022-08-15"
-                               },
-                               "tags": [ "eu" ],
-                               "count": 3
+                              "_id": "1",
+                              "location": "New York",
+                              "race": {
+                                "competitors": 100,
+                                "start_date": "2022-08-15"
+                              },
+                              "tags": [
+                                "eu"
+                              ],
+                              "count": 3
                             }
                           ],
                           "count": 1,
                           "status": {
                             "upsertedId": "1",
                             "matchedCount": 0,
-                            "modifiedCount": 1,
-                          }
+                            "modifiedCount": 1
+                          },
+                          "errors": [
```

Comment:
In this case, wouldn't we have `matchedCount` of `1`, if that one update failed? Or is the failed update counted in `modifiedCount`?

Reply:
I just added, to show what are all the possible output fields are returned. It doesn't corresponds to one specific case. Should I be adding multiple samples for different outcomes?

Reply:
Ok was wondering if that was the reason. Also realized that it may not be possible to show all combinations and keep it legal (insert for upsert only possible when no matches).
But since number of permutations can be pretty big, I think this is ok, just trying to keep things consistent where possible, but otherwise show kinds of things that can exist (even if not in specific combination).